src/linalg/nalgebra_bindings.rs

```diff
@@ -955,7 +955,7 @@ mod tests {
     #[test]
     fn pow_mut() {
         let mut a = DMatrix::from_row_slice(1, 3, &[1., 2., 3.]);
-        a.pow_mut(3.);
+        BaseMatrix::pow_mut(&mut a, 3.);
```

Comment:
BaseVector::pow_mut now conflicts with pow_mut from nalgebra

https://github.com/dimforge/nalgebra/blob/984bb1a63943aa68b6f26ff4a6acf8f68b833b70/src/linalg/pow.rs#L24

It has different arguments (integer) and it returns a Result<_> in nalgebra.